Essentials/src/main/resources/messages.properties

```
@@ -832,6 +832,7 @@ noPlacePermission=\u00a74You do not have permission to place a block near that s
 noPotionEffectPerm=\u00a74You do not have permission to apply potion effect \u00a7c{0} \u00a74to this potion.
 noPowerTools=\u00a76You have no power tools assigned.
 notAcceptingPay=\u00a74{0} \u00a74is not accepting payment.
+notAllowedToLocal=\u00a74You are not authorized to speak in local chat.
```

Maybe out of scope, but this might be nicer if we reworded these source strings along with the question/shout ones:
```suggestion
notAllowedToLocal=\u00a74You don't have permission to speak in local chat.
```